src/doc/reference.md

```diff
@@ -1271,15 +1271,16 @@ guaranteed to refer to the same memory address.
 
 Constant values must not have destructors, and otherwise permit most forms of
 data. Constants may refer to the address of other constants, in which case the
-address will have the `static` lifetime. The compiler is, however, still at
+address will have the `static` lifetime. (See below on [static lifetime
```

Comment:
How about "will have elided lifetimes where applicable, otherwise – in most cases – defaulting to the `'static` lifetime.